stockflux-securities-master/src/components/SecuritiesTable.js

```diff
@@ -48,7 +48,9 @@ const SecuritiesTable = () => {
           <Components.LargeSpinner />
         </div>
       ) : errorMessage ? (
-        <ErrorMessage errorMessage={errorMessage} />
+        <div className="securities-error-message-container">
```

Comment:
See above regarding renaming this container

Reply:
Yep done